benchopt/plotting/html/static/result.js

```diff
@@ -975,6 +979,18 @@ const show = (HTMLElements, style = 'initial') => {
   HTMLElements.forEach(h => h.style.display = style);
 };
 
+const scatterPlotHasMoreThan2PositiveValues = () => {
+  let res = true
+
+  getSolvers().forEach(solver => {
+    let isLogScaleAvailable = data(solver).scatter.is_log_scale_available
+
+    res = res && isLogScaleAvailable
```

Comment:
Is it a `and` or a `or`? If you have one curve that is positive, we could want to have a logscale no?